src/stores/contactmoment/index.ts

```diff
@@ -94,11 +103,19 @@ export const useContactmomentStore = defineStore("contactmoment", {
     } as ContactmomentenState;
   },
   getters: {
-    klantVoorHuidigeVraag(state): Klant | undefined {
+    klantVoorHuidigeVraag(state): ContactmomentKlant | undefined {
       return state.huidigContactmoment?.huidigeVraag.klanten
         ?.filter((x) => x.shouldStore)
         ?.map((x) => x.klant)?.[0];
     },
+
+    canStoreContactmoment(state): boolean {
+      return !state.huidigContactmoment?.vragen.some(
+        (vraag) =>
+          vraag.resultaat === "Contactverzoek gemaakt" &&
+          !vraag.klanten.some((klant) => klant.shouldStore)
```

Comment:
I think this needs to check if the Klant has contactInformation too.

Reply:
Nice catch.